jest/utils/index.ts

```diff
@@ -0,0 +1,8 @@
+import { fireEvent } from '@testing-library/react';
+export * from '@testing-library/react';
+
+export function fireClickAndMouseEvents(element: HTMLElement) {
```

Comment:
out of curiosity, wouldn't this be a double click? If you are **just** using it for the `Dropdown` component, just letting you know that the Dropdown just needs `mouseDown` 😅 

Also, I'd put  this in a `helpers` file (in my mind `index.ts` files only export)

Reply:
That part was intended and picked from Reach UI as the API requires a bit of tweaking in order to work.
https://github.com/reach/reach-ui/blob/develop/packages/menu-button/__tests__/menu-button.test.tsx#L2
I'll fix the helpers :)

Reply:
Oh cool! Ok then 😄